gymnasium/envs/mujoco/ant_v5.py

```diff
@@ -0,0 +1,427 @@
+
+    1. Any of the state space values is no longer finite
+    2. The z-coordinate of the torso is **not** in the closed interval given by `healthy_z_range` (defaults to [0.2, 1.0])
+
+    #### truncation
+    The maximum duration of an episode is 1000 timesteps.
+
+
+    ## Arguments
+    `gymnasium.make` takes additional arguments such as `xml_file`, `ctrl_cost_weight`, `reset_noise_scale`, etc.
+
+    ```python
+    import gymnasium as gym
+    env = gym.make('Ant-v5', ctrl_cost_weight=0.5, ...)
+    ```
+
+    | Parameter                                  | Type       | Default      |Description                    |
+    |--------------------------------------------|------------|--------------|-------------------------------|
+    |`xml_file`                                  | **str**    | `"ant.xml"`  | Path to a MuJoCo model |
+    |`forward_reward_weight`                     | **float**  | `1`          | Weight for _forward_reward_ term (see section on reward)|
+    |`ctrl_cost_weight`                          | **float**  | `0.5`        | Weight for _ctrl_cost_ term (see section on reward) |
+    |`contact_cost_weight`                       | **float**  | `5e-4`       | Weight for _contact_cost_ term (see section on reward) |
+    |`healthy_reward`                            | **float**  | `1`          | Weight for _healthy_reward_ term (see section on reward) |
+    |`main_body`                                 |**str|int** | `1`("torso") | Name or ID of the body, whose displacement is used to calculate the *dx*/_forward_reward_ (useful for custom MuJoCo models)|
+    |`terminate_when_unhealthy`                  | **bool**   | `True`       | If true, issue a done signal if the z-coordinate of the torso is no longer in the `healthy_z_range` |
+    |`healthy_z_range`                           | **tuple**  | `(0.2, 1)`   | The ant is considered healthy if the z-coordinate of the torso is in this range |
+    |`contact_force_range`                       | **tuple**  | `(-1, 1)`    | Contact forces are clipped to this range in the computation of *contact_cost* |
+    |`reset_noise_scale`                         | **float**  | `0.1`        | Scale of random perturbations of initial position and velocity (see section on Starting State) |
+    |`exclude_current_positions_from_observation`| **bool**   | `True`       | Whether or not to omit the x- and y-coordinates from observations. Excluding the position can serve as an inductive bias to induce position-agnostic behavior in policies |
+    |`include_cfrc_ext_in_observation`           | **bool**   | `True`       | Whether to include *cfrc_ext* elements in the observations. |
+    |`use_contact_forces` (`v4` only)            | **bool**   | `False`      | If true, it extends the observation space by adding contact forces (see `Observation Space` section) and includes contact_cost to the reward function (see `Rewards` section) |
+
+    ## Version History
```

Comment:
We need a v5 note here, if there are too many items, we could link to the blog post that provides this in more detail

Reply:
I will add the changelog (by copy and pasting from the changelog from the opening thread of the PR), after the rest of the review is done